packages/config/config/devices/0x031e/lzw36.json

```diff
@@ -386,31 +386,77 @@
 			]
 		},
 		"24[0xff0000]": {
-			"label": "Light LED Strip Effect Duration",
+			"label": "Light LED Effect Duration",
 			"description": "Light LED Strip Effect Duration. 1 to 60 = seconds, 61 to 120 minutes, 121 - 254 = hours, 255 = Indefinitely Default: 0",
 			"valueSize": 4,
 			"minValue": 0,
 			"maxValue": 255,
-			"defaultValue": 0,
+			"defaultValue": 255,
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"24[0xff00]": {
-			"label": "Light LED Strip Effect Intensity",
-			"description": "Light LED Strip Effect Intensity. 0 to 9. 0 = dim, 9 = bright Default: 3",
+			"label": "Light LED Effect Brightness",
+			"description": "Light LED Strip Effect Brightness. 0 to 10. 0 = dim, 10 = bright Default: 3",
 			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 9,
+			"maxValue": 10,
 			"defaultValue": 3,
 			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": true
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "10%",
+					"value": 1
+				},
+				{
+					"label": "20%",
+					"value": 2
+				},
+				{
+					"label": "30%",
+					"value": 3
+				},
+				{
+					"label": "40%",
+					"value": 4
+				},
+				{
+					"label": "50%",
+					"value": 5
+				},
+				{
+					"label": "60%",
+					"value": 6
+				},
+				{
+					"label": "70%",
+					"value": 7
+				},
+				{
+					"label": "80%",
+					"value": 8
+				},
+				{
+					"label": "90%",
+					"value": 9
+				},
+				{
+					"label": "100%",
+					"value": 10
+				}
+			]
 		},
 		"24[0xff]": {
-			"label": "Light LED Strip Effect Color",
+			"label": "Light LED Effect Color",
 			"description": "Light LED Strip Effect Color. Color - 0 - 255. Hue color wheel. value/255 * 360 = Hue color wheel value Range: 0-255 Default: 0",
```

Comment:
While we're cleaning up, this description should be simplified - it is a bit redundant in itself. Please no range/default in descriptions.

Please also check the other parameters that are similar.